README.md

```diff
@@ -49,12 +49,9 @@ In addition to the main purpose of testing the Rancher2 provider, `tfp-automatio
 ##### When testing locally, the following environment variables should be exported:
 ```yaml
 export RANCHER2_PROVIDER_VERSION=""                                     # Required
-export AWS_PROVIDER_VERSION=""                                          # Required for custom cluster / infrastructure building
-export LINODE_PROVIDER_VERSION=""                                       # Required for infrastructure building using Linode
-export HARVESTER_PROVIDER_VERSION=""                                    # Required for infrastructure building using Harvester
+export CLOUD_PROVIDER_VERSION=""                                          # Required for custom cluster / infrastructure building
```

Comment:
Would you be able to line up the comment (`# Required for...`) in line with the other comments?  it looks like there are 2 extra spaces before the comment begins 🙂